src/models/user/base-user.ts

```diff
@@ -1,3 +1,4 @@
+// Add to review
```

Comment:
Le entità del dominio sono Object e in quanto object hanno business logic, quindi sono classi concrete con metodi accessori e hanno metodi che fanno BL.
Mentre DTO e Model sono data structure e quindi non avranno mai BL all'interno.